exports.errorMessage = 'Something went wrong, Please try again later.';
exports.recordsFound = 'Records found.';
exports.recordsNotFound = 'Records not found.';
exports.invalidToken = 'Invalid Token, please enter valid token.';
exports.userIdRequired = 'UserId is required.';
exports.invalidUserId = 'Invalid UserId, Please use valid UserId.';
exports.tokenGeneratedSuccessfully = 'Token Generated successfully.';
exports.noToken = 'No token provided, Please provide token.';
exports.alertMessage = {
    users: {
        createSuccess: 'User saved successfully',
        createError: 'Error while creating user, Please try again later',
        updateSuccess: 'User updated successfully',
        updateError: 'Error while updating user, Please try again later',
        deleteSucces: 'User deleted successfully',
        deleteError: 'Error while deleting user, please try again later',
        listSuccess: 'User record(s) found successfully',
        listError: 'User record(s) not found, please try again later',
        internalError: 'Internal error',
    }
}